index.html

```diff
@@ -370,15 +369,16 @@ <h3 class="contact-title">
                 required
               />
             </li>
-            <li>
-              <label for="mail"></label>
+            <li class="input-container">
+              <label for="email"></label>
```

Comment:
The form accepts email with capital letters 
Kindly fix that

Reply:
could you please retry again I checked and the contact form doesn't accept email with uppercase letters, and more than that it print an alert to add a valid email format. 
![Capture](https://user-images.githubusercontent.com/80895497/127558038-c553e501-f599-4789-9862-ea959e12ecfd.PNG)

Reply:
The problem was that I didn't change my Live demo source, thanks for the heads-up, 👍